starter/src/app/app.module.ts

```diff
@@ -80,7 +81,6 @@ import { NoContentComponent } from "./no-content";
 /* tslint:disable:no-import-side-effect */
 // load SASS styles
 import "../styles/styles.scss";
-/* tslint:enable */
```

Comment:
Don't remove this `tslint:enable`.... otherwise TSLint will be disabled for the whole file :s